feedback-module/src/components/Module.js

```diff
@@ -100,6 +114,8 @@ function Module({ pagetitle, endpoint, dir }) {
       focusFirstError();
     } else {
       updateOtherField();
+      pageTitleAsScreen(en(screen.title));
+      pageChange(en(screen.title), en(SCREENS[nextScreen].title));
```

Comment:
Should be `en(screen.title, { page: pagetitle })` (same for other one) bc of the string interpolation in our translation config.